cranelift/module/src/module.rs

```diff
@@ -470,6 +470,13 @@ pub trait Module {
         &mut self,
         func: FuncId,
         bytes: &[u8],
+        relocs: &[(
+            binemit::CodeOffset,
+            ir::SourceLoc,
+            binemit::Reloc,
+            ir::ExternalName,
+            binemit::Addend,
+        )],
```

Comment:
I think the tuple should be a new type. I think `cranelift-simplejit` already has a type for this.

Reply:
Seems reasonable to me!